modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -238,11 +257,13 @@ final void readBinary(int bssVersion, DataInputStream is, String[] strings)
     private String[] stringStore;
     final String[] getStringStore() { return stringStore; }
 
-    /** Loads a binary stylesheet from a URL.
-     * @param url the URL from which the stylesheet will be loaded.
-     * @return the loaded stylesheet.
-     * @throws java.io.IOException if the binary stream corresponds to a more recent binary
-     * css version or if any other IO error occurs while reading from the stream.
+    /**
+     * Loads a binary stylesheet from a URL.
+     *
+     * @param url the URL from which the {@code Stylesheet} will be loaded
+     * @return the loaded {@code Stylesheet}
+     * @throws IOException if the binary stream corresponds to a more recent binary
+     * css version or if an I/O error occurs while reading from the stream
```

Comment:
Missing a `.` at the end of `@throws` description.

Reply:
Typically we don't end with a period for `@param`, `@return`, or `@throws`. I recommend leaving this one as is.